src/reflect/value.go

```diff
@@ -11,6 +12,8 @@ type valueFlags uint8
 // contained in an interface{} directly, like whether this value was exported at
 // all (it is possible to read unexported fields using reflection, but it is not
 // possible to modify them).
+//
+// These flags are shared with the internal/reflectlite package.
 const (
 	valueFlagIndirect valueFlags = 1 << iota
```

Comment:
If these are shared, should these constants `=` their `reflectlite` counterparts?